frontendGraphApi/src/main/java/com/virtuslab/gitmachete/frontend/graph/api/labeling/SyncToRemoteStatusLabelGenerator.java

```diff
@@ -17,8 +17,8 @@ private SyncToRemoteStatusLabelGenerator() {}
   public static String getLabel(SyncToRemoteStatus.Relation relation, String remoteName) {
     return Match(relation).of(
         Case($(Untracked), "untracked"),
-        Case($(Ahead), "ahead of " + remoteName),
-        Case($(Behind), "behind " + remoteName),
+        Case($(AheadOfRemote), "ahead of " + remoteName),
+        Case($(BehindOfRemote), "behind of " + remoteName),
```

Comment:
```suggestion
        Case($(BehindRemote), "behind " + remoteName),
```

`behind` + `of` is rarely (if ever) used in English :)

Reply:
ahh.. :man_facepalming:  alright